src/test/java/model/OperationTest.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OperationTest {
+    private static final String INVALID_OPERATION = "?";
```

Comment:
no need to test enum, add it to exclude of jacoco

Reply:
added to `exclude` in pom.xml